src/main/java/mate/academy/dao/impl/MovieDaoImpl.java

```diff
@@ -45,6 +38,10 @@
 
     @Override
     public List<Movie> getAll() {
-        return null;
+        try (Session session = HibernateUtil.getSessionFactory().openSession()) {
+            return session.createQuery("from Movie ", Movie.class).list();
+        } catch (Exception e) {
```

Comment:
Consider logging the exception `e` here for debugging purposes before throwing the `DataProcessingException`. This can help in tracing issues during runtime.